packages/bluesnap-direct-integration/src/BlueSnapDirectPayByBankPaymentMethod.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FunctionComponent } from 'react';
+
+import {
+    PaymentMethodProps,
+    PaymentMethodResolveId,
+    toResolvableComponent,
+} from '@bigcommerce/checkout/payment-integration-api';
+import { Fieldset, Legend } from '@bigcommerce/checkout/ui';
+
+import { isBlueSnapDirectInitializationData } from './BlueSnapDirectInitializationData';
+import BlueSnapDirectTextField from './fields/BlueSnapDirectTextField';
+
+const BlueSnapDirectPayByBankPaymentMethod: FunctionComponent<PaymentMethodProps> = ({
+    method,
+    language,
+}) => {
+    if (!isBlueSnapDirectInitializationData(method.initializationData)) {
+        throw new Error('Unable to get initialization data');
+    }
+
+    return (
+        <Fieldset
+            legend={
+                <Legend hidden>{language.translate('payment.bluesnap_direct_iban.label')}</Legend>
+            }
+            style={{ paddingBottom: '1rem' }}
+        >
+            <BlueSnapDirectTextField
```

Comment:
should it be just one input? do we need some permission checkbox or something?

Reply:
no permission box needed, just one input field for IBAN